config/manager/kustomization.yaml

```diff
@@ -3,3 +3,19 @@ resources:
 
 generatorOptions:
   disableNameSuffixHash: true
+
+configMapGenerator:
+- name: codeflare-stack-config
```

Comment:
I think that `stack-config` could be enough.
All resources are automatically prefixed with `codeflare-operator-` - https://github.com/project-codeflare/codeflare-operator/blob/main/config/default/kustomization.yaml#L9